test/distributed/org/apache/cassandra/fuzz/sai/SingleNodeSAITestBase.java

```diff
@@ -193,33 +232,50 @@ private void basicSaiTest(EntropySource rng, SchemaSpec schema)
             if (rng.nextFloat() > 0.9995f)
                 history.deletePartition(partitionIndex);
 
-            if (i % REPAIR_SKIP == 0)
-                history.custom(() -> repair(schema), "Repair");
-            else if (i % FLUSH_SKIP == 0)
+            if (i % FLUSH_SKIP == 0)
                 history.custom(() -> flush(schema), "Flush");
             else if (i % COMPACTION_SKIP == 0)
                 history.custom(() -> compact(schema), "Compact");
+            else if (i % repairSkip == 0)
+                history.custom(() -> repair(schema), "Repair");
 
-            if (i > 0 && i % 1000 == 0)
+            if (i > 0 && i % VALIDATION_SKIP == 0)
             {
-                for (int j = 0; j < 5; j++)
+                for (int j = 0; j < QUERIES_PER_VALIDATION; j++)
```

Comment:
why go from `5` to `8`?

Reply:
Number of partitions increased, so wanted to bump this as well.